app/views/RoomsListView/index.tsx

```diff
@@ -91,53 +144,27 @@ const sortPreferencesShouldUpdate = ['sortBy', 'groupByType', 'showFavorites', '
 
 const displayPropsShouldUpdate = ['showAvatar', 'displayMode'];
 
-const getItemLayout = (data, index, height) => ({
+const getItemLayout = (data: ISubscription[] | null | undefined, index: number, height: number) => ({
```

Comment:
data?: ISubscription[] | null

Reply:
In this case I added as undefined because the return type of `getItemLayout` from `FlatList` could be this 3 options
![Screen Shot 2022-02-24 at 11 35 20](https://user-images.githubusercontent.com/14894380/155544648-d110f606-d95b-4c22-b115-46ce1d4f7f2c.png)

And, the first param of a function can not be optional.